integration-tests/deployment/ccip/changeset/2_initial_deploy.go

```diff
@@ -14,18 +16,14 @@ func Apply0002(env deployment.Environment, c ccipdeployment.DeployCCIPContractCo
 	ab, err := ccipdeployment.DeployCCIPContracts(env, c)
 	if err != nil {
 		env.Logger.Errorw("Failed to deploy CCIP contracts", "err", err, "addresses", ab)
-		return deployment.ChangesetOutput{}, err
+		return deployment.ChangesetOutput{}, deployment.MaybeDataErr(err)
 	}
 	js, err := ccipdeployment.NewCCIPJobSpecs(env.NodeIDs, env.Offchain)
 	if err != nil {
 		return deployment.ChangesetOutput{}, err
 	}
-	proposal, err := ccipdeployment.GenerateAcceptOwnershipProposal(env, env.AllChainSelectors(), ab)
```

Comment:
This can be its own changeset, probably too much to deploy all the contracts and accept ownership in the same one